pkg/pipelines/bootstrap.go

```diff
@@ -27,30 +32,20 @@ func Bootstrap(quayUsername, baseRepo, prefix string) error {
 
 	outputs := make([]interface{}, 0)
 
-	tokenPath, err := pathToDownloadedFile("token")
-	if err != nil {
-		return fmt.Errorf("failed to generate path to file: %w", err)
-	}
-	f, err := os.Open(tokenPath)
-	if err != nil {
-		return err
-	}
-	defer f.Close()
-
-	githubAuth, err := createOpaqueSecret("github-auth", f)
+	githubAuth, err := createOpaqueSecret("github-auth", githubToken)
 	if err != nil {
 		return err
 	}
 	outputs = append(outputs, githubAuth)
 
-	authJSONPath, err := pathToDownloadedFile(quayUsername + "-auth.json")
+	authJSONPath, err := homedir.Expand(quayIOAuthFilename)
 	if err != nil {
 		return fmt.Errorf("failed to generate path to file: %w", err)
 	}
 
-	f, err = os.Open(authJSONPath)
+	f, err := os.Open(authJSONPath)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to open path authJSON : %w", err)
```

Comment:
With this, it'll output the filename it tries to open, and the error.

```suggestion
		return fmt.Errorf("failed to open path %s: %w", authJSONPath, err)
```

Reply:
I will make this change